File: wwwroot/frontend/src/Modele/Joueur.ts
import Equipe from "./Equipe";
import Pays from "./Pays";

export default interface Joueur{
    id:number,
    nom:string,
    prenom:string,
    numero:number,
    equipeId:number,
    equipe:Equipe,
    paysId:number,
    pays:Pays
}

File: Models/Joueur.cs
using System.ComponentModel.DataAnnotations;

namespace SuperBowlWeb.Models
{
    public class Joueur
    {
        [Key]
        public long Id { get; set; }

        public string Nom { get; set; }

        public string Prenom { get; set; }

        public short Numero { get; set; }

        public int? EquipeId { get; set; }
        public Equipe Equipe { get; set; }

        public int PaysId { get; set; }
        public Pays Pays { get; set; }
    }
}
